app/src/main/res/values/dimens.xml

```diff
@@ -29,4 +29,10 @@
     <dimen name="removable_image_layout_margin">13dp</dimen>
 
     <dimen name="entry_name_icon_margin">28dp</dimen>
+    <dimen name="trophy_margin">28dp</dimen>
+
+    <dimen name="category_card_height">128dp</dimen>
+    <dimen name="category_card_width">325dp</dimen>
```

Comment:
can you check with the designers on this? They may want this to fill the space instead of being a fixed width.